lib/chef/knife/vra_server_create.rb

```diff
@@ -46,6 +46,11 @@ class VraServerCreate < ServerCreateCommand
                long:        '--requested-for LOGIN',
                description: 'The login to list as the owner of this resource. Will default to the vra_username parameter'
 
+        option :server_create_timeout,
```

Comment:
Where is this option used?  I can't find any reference to it in your PR or in knife-cloud.